docs/sphinx/source/contributing/index.rst

```diff
@@ -0,0 +1,12 @@
+============
+Contributing
+============
+
+
```

Comment:
What about moving the introduction to this place and guide which place is the best for the user? Expecting other people to weight in, I don't dislike how it is now either. In the big wall of the review I share how I would re-distribute the documentation.